submissions/vasylpaliy/html_css_popup/index.html

```diff
@@ -0,0 +1,153 @@
+         <nav>
+            <ul class="container-nav">
+               <li>
+                  <a href="#" class="container-nav-link ">Mail</a>
+               </li>
+               <li>
+                  <a href="#" class="container-nav-link ">Images</a>
+               </li>
+               <li>
+                  <input type="checkbox" id="my-popup" class="input-nav-popup">
+                  <label for="my-popup" class="container-label">
+                  <img src="icons/popup-button.png" alt="Open popup menu" class="container-navbar-img">
+                  </label>
+                  <div class="popup-container">
+                     <ul class="popup">
+                        <li>
+                           <a href="#" class="popup-text ">
+                              <img src="icons/calendar.png" alt="Calendar" class="popup-img">
+                              <p class="popup-p">Calendar</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/mail.png" alt="Mail" class="popup-img">
+                              <p class="popup-p">Mail</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/news.png" alt="News" class="popup-img">
+                              <p class="popup-p">News</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/maps.png" alt="Map" class="popup-img">
+                              <p class="popup-p">Map</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/files.png" alt="Files" class="popup-img">
+                              <p class="popup-p">Files</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/contacts.png" alt="Contacts" class="popup-img">
+                              <p class="popup-p">Contacts</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/tanslation.png" alt="Translate" class="popup-img">
+                              <p class="popup-p">Translate</p>
+                           </a>
+                        </li>
+                        <li>
+                           <a href="#" class="popup-text">
+                              <img src="icons/videos.png" alt="Videos" class="popup-img">
```

Comment:
Something going wrong because all icons have issues with aspect ratio, please fix it.
![image](https://user-images.githubusercontent.com/16196199/187019343-ca520d09-f3dc-46d5-a39c-d2f852351e66.png)